server/src/main/java/org/elasticsearch/cluster/metadata/IndexMetadata.java

```diff
@@ -2155,6 +2209,12 @@ public static void toXContent(IndexMetadata indexMetadata, XContentBuilder build
             indexMetadata.timestampRange.toXContent(builder, params);
             builder.endObject();
 
+            if (context != Metadata.XContentContext.API && indexMetadata.writeLoad != null) {
```

Comment:
I think we would like to see this in the `_cluster/state` API for debugging purpose.

```suggestion
            if (indexMetadata.writeLoad != null) {
```